src/extensions/default/Inline3DParametersEditor/main.js

```diff
@@ -0,0 +1,87 @@
+define(function (require, exports, module) {
+    "use strict";
+
+    var EditorManager           = brackets.getModule("editor/EditorManager"),
+        ExtensionUtils          = brackets.getModule("utils/ExtensionUtils"),
+        InlineParameterEditor   = require("InlineParameterEditor").InlineParameterEditor,
```

Comment:
Can you re-order to separate what you're loading from your extension with `require` vs. what is being loaded from Brackets core with `brackets.getModule`?  It makes it a bit easier to understand the code when they are grouped.